cmd/kubeadm/app/phases/upgrade/prepull.go

```diff
@@ -177,11 +179,11 @@ func buildPrePullDaemonSet(component, image string) *apps.DaemonSet {
 				Spec: v1.PodSpec{
 					Containers: []v1.Container{
 						{
-							Name:    component,
-							Image:   image,
-							Command: []string{"/bin/sleep", "3600"},
```

Comment:
@RA489 
this has to be replaced with something that waits for the DaemonSet to prepull.

Reply:
sure. Thanks! for the review.

Reply:
So, another possibility is to invoke something non-existent here and let the pods crash loop. That's the only distroless-safe idea using DaemonSet that comes to my mind. However, that idea is plainly wrong in my opinion.
Hence, we should just opt in for the ImagePullCheck upon node upgrade.